source/_integrations/iammeter.markdown

```diff
@@ -15,6 +15,11 @@ ha_integration_type: integration
 ---
 
 [IAMMETER](https://www.iammeter.com/) provides both a bi-directional single-phase energy meter([WEM3080](https://www.iammeter.com/products/single-phase-meter)) and a bi-directional three-phase energy monitor ([WEM3080T](https://www.iammeter.com/products/three-phase-meter)). Both of them can be integrated into Home Assistant.
+[WEM3050T](https://www.iammeter.com/products/3phase-meter-3050t): latest 3phase/split phase meter, released by IAMMETER on Nov 2023.
+WEM3050T provides nearly all of the features of WEM3080T except the cloud service.
```

Comment:
```suggestion
The WEM3050T provides nearly all of the features the WEM3080T provides, with the exception of the cloud service..
```

Reply:
If I'm reading the GitHub UI correctly seems like there's an extra . there?

Reply:
thank you for the catch